app/templates/administrative-units/administrative-unit/local-involvements/edit.hbs

```diff
@@ -0,0 +1,71 @@
+{{page-title "Bewerk"}}
+
```

Comment:
I forgot the wrapping div with the scroll class which is probably needed on this page.

Reply:
Ok